src/inventory_ui.cpp

```diff
@@ -1286,10 +1328,22 @@ std::pair<std::string, nc_color> inventory_selector::get_footer( navigation_mode
 
 void inventory_selector::draw_footer( WINDOW *w ) const
 {
+    int filter_offset = 0;
+    if( has_available_choices() || !filter.empty() ) {
+        std::string text = string_format( filter.empty() ? _("[%s]Filter") : _("[%s]Filter: "),
+                                          ctxt.press_x( "INVENTORY_FILTER", "", "", "" ) );
+        filter_offset = utf8_width( text + filter ) + 6;
+
+        mvwprintz( w, getmaxy( w ) - border, 2, c_ltgray, "< " );
+        wprintz( w, c_ltgray, text.c_str() );
```

Comment:
Native curses functions with variadic arguments like these should not be called with user-defined string as format argument. Consider what happens when `text` contains "%s" - `wprintz` will assume there's another argument of type `const char*`. Instead call them as `wprintz( ..., "%s", text.c_str() )`. (Calling them with a string literal like below that is safe as you can guarantee it doe not contain any format specifiers.)